apps/cowswap-frontend/src/legacy/state/price/updater.ts

```diff
@@ -79,25 +92,43 @@ function quoteUsingSameParameters(currentParams: FeeQuoteParams, quoteInfo: Quot
     hasSameReceiver
   // 2 checks: if there's a quoteInfo user address (meaning quote was already calculated once) and one without
   // in case user is not connected
-  return userAddress ? currentUserAddress === userAddress && paramsWithoutAddress : paramsWithoutAddress
+
+  const sameParams = userAddress ? currentUserAddress === userAddress && paramsWithoutAddress : paramsWithoutAddress
```

Comment:
Interesting, why do we check only `userAddress` when the wallet is connected?

Reply:
`isRefetchQuoteRequired` looks overcomplicated, what if we just delete it? :)
Maybe we can just do polling every 30 seconds without any checks?